homeassistant/components/prometheus/__init__.py

```diff
@@ -668,6 +680,69 @@ def _handle_humidifier(self, state: State) -> None:
                     self._labels(state, {"mode": mode}),
                 ).set(float(mode == current_mode))
 
+    def _handle_water_heater(self, state: State) -> None:
+        # Temperatures
+        self._handle_climate_temp(
```

Comment:
Now when `self._handle_climate_temp` is used for domains other than `climate`, it might be better to rename it to smth like `_handle_temp`, or maybe even `_temperature_metric`, since it's just a wrapper around `self._metric`, and "temp" might be a bit ambiguous (could also mean "temporary"). WDYT?